_api-reference/document-apis/get-documents.md

```diff
@@ -48,6 +108,79 @@ _source_includes | String | A comma-separated list of source fields to include i
 version | Integer | The version of the document to return, which must match the current version of the document.
 version_type | Enum | Retrieves a specifically typed document. Available options are `external` (retrieve the document if the specified version number is greater than the document's current version) and `external_gte` (retrieve the document if the specified version number is greater than or equal to the document's current version). For example, to retrieve version 3 of a document, use `/_doc/1?version=3&version_type=external`.
 
+### Realtime
```

Comment:
Why is the information repeated here - shouldn't this be in format similar to above older query parameters ?

Reply:
@bharath-techie: This is the same information. It actually belongs in this section sense it pertains more to the Request options. Deleted the other section.